tests/testthat/test-horvitz-thompson.R

```diff
@@ -184,6 +205,48 @@ test_that("Horvitz-Thompson properly checks arguments", {
     "N|declaration"
   )
 
+  # Reserved variable names
+  dat$`.clusters_ddinternal` <- 1
```

Comment:
For internal type things, also consider making them attributes instead of list elements. This will make pretty printing, iteration, etc easier